code/zoltar_scripts/README.md

```diff
@@ -0,0 +1,36 @@
+We use Zoltar to store forecasts and metadata. This means all forecasts and metadata can be accessed via web download or programatically using R or Python.
+
+- [View the ECDC European COVID-19 Forecast Hub on Zoltar](https://www.zoltardata.com/project/238)
+- [Find out more about Zoltar](https://docs.zoltardata.com/)
+
+----
+
+### Hub developers
+
+#### Local upload to Zoltar
+
+First ensure you have set up  hub Zoltar access by adding `Z_USERNAME` and `Z_PASSWORD` as environment variables.
+
+```
+# Clone repo with validations submodule
+git clone epiforecasts/covid19-forecast-hub-europe --recurse-submodules
+
+# Set hub as working directory, e.g.
+# cd Github/covid19-forecast-hub-europe
+
+# Detect modified files, convert to json and upload
+python code/zoltar_scripts/upload_zoltar.py
+```
+Detection, validation, and upload takes ~10 seconds per forecast.
+
+#### Other tasks
+
+ - Compare forecasts in the repo with those in Zoltar with:
+```
+ python code/zoltar_scripts/compare_repo_to_zoltar.py
+ ```
+
+- Create json file with:
```

Comment:
What's the purpose of this?

Reply:
It's unlikely to ever be used (hopefully), but if for some reason the python-zoltar connection is not working, then this script creates the correctly-formatted json file which can be uploaded manually using the zoltar web interface. I've added this to the README.